rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/facade/copy.kt

```diff
@@ -285,7 +253,7 @@ class CopyBlocks : Iterable<CopyBlock> {
                     absoluteLine - block.start
                 } else {
                     val childrenBefore = children.filter { child -> child.end <= absoluteLine }
-                    absoluteLine - block.start - childrenBefore.sumBy { childBefore -> childBefore.end - childBefore.start } + 1
+                    absoluteLine - block.start - childrenBefore.sumOf { childBefore -> childBefore.end - childBefore.start } + 1
```

Comment:
One more thing, why have you replaced `sumBy` with `sumOf?`

Reply:
Beacause in the last kotlin versions sumBy seem to be deprecated:

```
@Deprecated
@DeprecatedSinceKotlin
public inline fun <T> Iterable<T>.sumBy(
    selector: (T) -> Int
): Int
```